src/Libraries/RevitNodes/Elements/FamilyInstance.cs

```diff
@@ -492,12 +492,50 @@ internal static FamilyInstance FromExisting(Autodesk.Revit.DB.FamilyInstance fam
 
         #endregion
 
+        #region Public Methods
+
         public override string ToString()
         {
             return InternalFamilyInstance.IsValidObject ? InternalFamilyInstance.Name : "empty";
         }
 
-        #region Public methods
+        /// <summary>
+        /// Get FamilyInstance Host Element
+        /// </summary>
+        public Element GetHost
+        {
+            get
+            {
+                if (this.InternalFamilyInstance.Host != null)
+                    return ElementWrapper.Wrap(this.InternalFamilyInstance.Host, true);
```

Comment:
I mean, what if you manually created something in Revit on top of a element that was created by Dynamo.

IE. someone creates a wall with Dynamo so this is Dynamo owned, then they create a window on this wall, and then call getHost on it?